vector/src/main/res/layout/item_display_read_receipt.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="44dp"
```

Comment:
I do not like hard-coded height on a cell containing text. If user choose to use big font, the render will not be correct.
Instead use `wrap_content` with padding on text for instance